### ISSUE TITLE
A11y - Required label, insufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#3736]
## Issue Description

Ensures the contrast between foreground and background colours meets WCAG 2 AA contrast ratio thresholds.

- **Actual**: 4.3:1
- **Expected**: 4.5:1

## Element Location

https://github.com/GCTC-NTGC/gc-digital-talent/blob/c2b51159237ca50bb903e3017c6663f0a3a1dcdd/frontend/common/src/components/inputPartials/InputLabel/InputLabel.tsx#L54-L67

## To solve this issue, you need to...

Fix the following:

- Element has insufficient colour contrast of 4.3 (foreground colour: #e90101, background colour: #f5f5f5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1

## Acceptance Criteria

- [ ] Update to have 4.5:1 colour contrast
- [ ] Write axe test for inputs

**Comment by gobyrne** [comment]
Might have been fixed with Hydrogen merge.

**Comment by JoshBeveridge** [comment]
I don't believe this was touched, but we can tack the `.dark` or `.darker` color modifiers to the Hydrogen attribute's color option to fix it.
